commands/project/dev.ts

```diff
@@ -107,12 +107,12 @@ exports.handler = async options => {
     (!hasPublicApps && isSandbox(accountConfig));
 
   // The account that the project must exist in
-  let targetProjectAccountId = derivedAccountId;
+  let targetProjectAccountId = providedAccountId ? derivedAccountId : null;
```

Comment:
Theoretically, if `providedAccountId` exists, it's the same thing as `derivedAccountId` right? Could this just be
`let targetProjectAccountId = providedAccountId`?

Reply:
`providedAccountId` is not the same as `derivedAccountId`. `providedAccountId` is just the raw value that the user supplies in the flag, which can include the name in the config. The middleware takes the raw value from `providedAccountId` and spits out `derivedAccountId`, which will always be the account ID (type number). 

Going to resolve this but Slack me with any more questions.